api/hub/src/datasources/moderation-report.ts

```diff
@@ -195,6 +195,26 @@ class ModerationReportAPI extends DataSource {
         logger.warn(`pending decision could not be created for contentID ${contentID}`);
         // throw new Error('pending decision could not be created');
       }
+      // send email notification to moderators
+      const text = `There is a new pending request for moderation.
+      Please visit the moderation app (https://akasha.ethereum.world/moderation-app/home)
+      to moderate this content.
+      \nThank you!`;
+      const html = `<p>There is a new pending request for moderation.
+      Please visit the <a href="https://akasha.ethereum.world/moderation-app/home">moderation app</a>
+      to moderate this content.</p>
+      <br>
+      <p>Thank you!</p>`;
+      try {
+        await sendEmailNotification(process.env.MODERATION_EMAIL,
```

Comment:
if you need to try catch then you can use directly the promise api `sendEmailNotification(...).then(_=>whatever).catch(err => do err stuff)`

Reply:
Done.